pkg/mkimage-iso-efi/initrd.sh

```diff
@@ -81,7 +87,17 @@ done
 if [ -n "$rootdev" ]; then
     echo "found root filesystem: $rootdev, switching"
     mount "$rootdev" /newroot
-    exec switch_root /newroot /sbin/init
+    # Now, check for the installer rootfs squashfs image
+    rootfsimg=/newroot/"$root_img"
```

Comment:
Is it worth check if `root_img` is blank? If it is, no need for the extra mount, just `switch_root /newroot`, like before.

Reply:
No, because the `if` right below will check for a regular file pointed by `$rootfsimg`, and it will fail if doesn't find it, we should always expect an image as of now, in case of failure we just drop to the shell, as it was before....

Reply:
honestly, here I think it will just add and extra check and complexity, but better safe then sorry, makes sense to try to switch root... I will change....

Reply:
done.